### DISCUSSION TITLE
Implementing with Delphi

### DISCUSSION THREAD

**Discussion by sergioschu** [open-telemetry/community#1774] [Community Discussions]
I would like to know if you know any material for implementing openTelemetry with Delphi, or if there is already a wrapper to be used?
I have an application and service in Delphi and would like to implement it?

**Comment by trask** [comment]
hi @sergioschu! I don't see any prior mention of Delphi (https://github.com/search?q=org%3Aopen-telemetry+delphi&type=issues). another place you could ask is on [#opentelemetry](https://cloud-native.slack.com/archives/CJFCJHG4Q) slack channel.
